scripts/webpack.conf.js

```diff
@@ -13,6 +13,20 @@ module.exports = Promise.all([
         'process.env.DEBUG': JSON.stringify(process.env.DEBUG || false),
       }),
     );
+    findPostCssRule:
+    for (const { use } of config.module.rules) {
+      if (!Array.isArray(use)) continue;
+      for (const { loader, options } of use) {
+        if (loader === 'postcss-loader') {
+          options.plugins.unshift(
+            require('postcss-custom-properties')({
+              preserve: false
+            }),
+          );
+          break findPostCssRule;
+        }
+      }
+    }
```

Comment:
Actually we don't have to do anything here, `postcss-custom-properties` is already supported in `precss`.
FYI postcss will respect `.browserslistrc`. If CSS variables are supported by all the browsers defined in `.browserslistrc`, they will no longer be transformed.

Reply:
I don't want to preserve the variables in the final CSS as they add overhead and don't provide any runtime benefits.

Reply:
In that case you may want sass-like syntax, see [precss](https://github.com/jonathantneal/precss). CSS variables are supposed to be preserved so that we can change theme with custom CSS easily by modifying the variables, without overriding CSS properties everywhere. I think here we should use sass variables for `px`s and CSS variables for colors. They are both supported in this project.

Reply:
Done.